lib/caffeine/engine.rb

```diff
@@ -3,6 +3,9 @@
 require 'caffeine/strategies/decent_exposure/pages_search_strategy'
 require 'caffeine/concerns/sluggable'
 require 'caffeine/concerns/seo_friendly'
+require 'rails-assets-angular'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.